app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_list = {}
+
+    def storage(*args, **kwargs) -> int:
+        if args not in cache_list.keys():
```

Comment:
Checking `if args not in cache_list.keys()` is unnecessary. You can directly check `if args not in cache_list:` which is more efficient.